client/my-sites/google-my-business/controller/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External Dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+
+import GMBSelectBusinessType from '../select-business-type';
+
+export function show( context, next ) {
+	const { params } = context;
+	context.primary = <GMBSelectBusinessType siteId={ params.site_id } />;
+	next();
+}
```

Comment:
Why is this controller located in a `controller` directory? Usually it's next to the routes definition, no?

Reply:
I've copied the structure from `my-sites/checklist`

Reply:
I don't think that we should follow this pattern, at least that's not something [our documentation promotes](https://github.com/Automattic/wp-calypso/blob/master/docs/guide/hello-world.md#adding-a-new-section).